packages/manager/modules/manager-pci-common/src/api/hook/useFlavors.ts

```diff
@@ -132,6 +132,25 @@ export const useMergedKubeFlavors = (projectId: string, region: string) => {
         const plan = availability.plans?.find(
           (_plan) => _plan.code === flavor.planCodes.hourly,
         );
+
+        // Bugfix: Issue with selecting the correct monthly price in the US
+        // -----------------------------------------------------------
+        // Context: Normally, we take the first element of the `pricings` array
+        // to determine the monthly price. However, for US offers, this first
+        // value may not be correct. Instead, we need to specifically look
+        // for the entry containing 'renew' in `capacities`.
+        //
+        // Solution:
+        // - We first search the `pricings` array for an entry where `capacities`
+        //   includes 'renew' and return it if found.
+        // - If no such entry exists, we fallback to the first element of `pricings` (if available).
+        const checkPriceMonthly = useMemo(
```

Comment:
You cannot invoke `useMemo` hook inside another `useMemo` function ;) (line 114)

BTW, as it's not a function I'd rather simply call it `monthlyPrice`